src/review/review.controller.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Param,
+  Patch,
+  Post,
+} from '@nestjs/common';
+import { ReviewService } from './review.service';
+import { CreateReviewDto } from './dto/create-review.dto';
+import { UpdateReviewDto } from './dto/update-review.dto';
+import { CurrentUser, User } from '@app/common';
+import {
+  ApiTags,
+  ApiOperation,
+  ApiResponse,
+  ApiBody,
+  ApiParam,
+} from '@nestjs/swagger';
+
+@ApiTags('reviews')
+@Controller('reviews')
+export class ReviewController {
+  constructor(private readonly reviewService: ReviewService) {}
+
+  @Post()
+  @HttpCode(HttpStatus.CREATED)
+  @ApiOperation({ summary: 'Create a new review' })
+  @ApiResponse({
+    status: HttpStatus.CREATED,
+    description: 'Review successfully created',
+  })
+  @ApiResponse({
+    status: HttpStatus.BAD_REQUEST,
+    description: 'Invalid input data',
+  })
+  @ApiBody({ type: CreateReviewDto })
+  async create(
+    @Body() createReviewDto: CreateReviewDto,
+    @CurrentUser() { userId }: User,
+  ) {
+    return this.reviewService.create(createReviewDto, userId);
+  }
```

Comment:
Delete the HTTP code. By default Nest knows to map the correct HTTP codes unless you need to specify it